MdeModulePkg/Universal/SetupBrowserDxe/IfrParse.c

```diff
@@ -1655,7 +1715,13 @@ ParseOpCodes (
           CurrentForm->SuppressExpression = (FORM_EXPRESSION_LIST *)AllocatePool (
                                                                       (UINTN)(sizeof (FORM_EXPRESSION_LIST) + ((ConditionalExprCount -1) * sizeof (FORM_EXPRESSION *)))
                                                                       );
-          ASSERT (CurrentForm->SuppressExpression != NULL);
+          // MU_CHANGE [BEGIN] - CodeQL change
+          if (CurrentForm->SuppressExpression == NULL) {

```

Comment:
Similar to some other places, if `CurrentForm->SuppressExpression` is `NULL` and a return happens, `CurrentForm` no longer needs to be allocated.

Reply:
Updated but please come sanity check my work to make sure I layered the frees and entry list removals correctly

Reply:
Looks okay